packages/blockchain-wallet-v4/src/redux/data/bch/sagas.js

```diff
@@ -86,7 +88,7 @@ export default ({ api }) => {
       yield put(A.fetchTransactionHistoryLoading())
       const currency = yield select(selectors.settings.getCurrency)
       if (address) {
-        const convertedAddress = fromCashAddr(address)
+        const convertedAddress = convertFromCashAddrIfCashAddr(address)
```

Comment:
I had forgotten to add this check to fetchTransactionHistory saga, so it was trying to convert an xpub to cashAddr